api/logs/v1/http.go

```diff
@@ -120,6 +120,34 @@ func NewHandler(read, tail, write *url.URL, opts ...HandlerOption) http.Handler
 				prometheus.Labels{"group": "logsv1", "handler": "query_range"},
 				proxyRead,
 			))
+			// Endpoints exposed by the querier and frontend
+			// See https://grafana.com/docs/loki/latest/api/#microservices-mode
+			r.Handle("/api/v1/label", c.instrument.NewHandler(
```

Comment:
AFAICS, `/api/v1/label` does not existing on Loki's side. Maybe a typo?

Reply:
It's not documented, but Grafana 7 sends a request to that API. It's present in the querier and query-frontend: https://github.com/grafana/loki/blob/v1.6.1/pkg/loki/modules.go#L333